akvo/templates/reports/grids.css

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
I'm agree with you. But because the `wkhtmltopdf` is not running under a web server, it can only do an external request to load resources (css, images, etc.). it's not able to resolve relative url (url without the hostname). 

I can't seem to find an elegant way to load the CSS like any other CSS are loaded in Django templates. Beside that, I think it's unnecessary to do an external request to load the internal CSS. That's why I'm using the Django template's `include` tag to embed the CSS, and separate the CSS file just for modularity purpose.